app/assets/stylesheets/dm/pages/_practice.scss

```diff
@@ -800,6 +800,27 @@ li.small-disc:before {
   }
 }
 
+/*********************************
+ EDITING GUIDE
+**********************************/
+
+#dm-editing-guide-modal {
+  .dm-editing-guide-modal-close {
+    background-color: inherit;
+    color: #2e2e2e;
```

Comment:
We should try to use the design system color palettes where we can. For example, this would look something like:
```
color: color($theme-color-base-darkest);
```

Here's the link to the palettes: https://designsystem.digital.gov/design-tokens/color/system-tokens/

Reply:
👀 Thanks! Super helpful to have this example.